dashboard/db/migrate/20160130084049_create_survey_results.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSurveyResults < ActiveRecord::Migration
+  def change
+    create_table :survey_results do |t|
+      t.references :user, index: true, foreign_key: true
+      t.text :properties, limit: 65535
+
+      t.timestamps null: false
+    end
+  end
```

Comment:
ActiveRecord knows that it can reverse create_table with drop_table so we don't have to tell it.